scripts/dockerbot/pytestbase/Dockerfile

```diff
@@ -1,11 +1,20 @@
-FROM fedora:23
-RUN dnf install -y gcc postgresql-devel libxml2-devel libxslt-devel zeromq3-devel git nano python-devel gnupg gnupg2 libcurl-devel redhat-rpm-config findutils libffi-devel openssl-devel
-RUN git clone https://github.com/RedHatQE/cfme_tests.git
-RUN dnf -y install python-setuptools; easy_install pip
-RUN pip install -U pip
-RUN cd /cfme_tests && PYCURL_SSL_LIBRARY=nss pip install -U -r /cfme_tests/requirements/dev.txt --no-cache-dir
+FROM fedora:25
+RUN dnf install -y \
+		gcc postgresql-devel libxml2-devel libxslt-devel zeromq3-devel git nano python-devel gnupg gnupg2 \
+		libcurl-devel redhat-rpm-config findutils libffi-devel openssl-devel python-virtualenv && dnf clean all
+
+RUN git config --global user.email "me@dockerbot" && git config --global user.name "DockerBot"
+
+RUN git clone https://github.com/ManageIQ/integration_tests
+RUN cd /integration_tests && \
+	python -m cfme.scripting.quickstart && \
+	python -m cfme.scripting.quickstart --mk-virtualenv ../cfme_pristine_venv
```

Comment:
once to make the virtualenv to be used in tests, once to make the virtualenv used in result submission